### ISSUE TITLE
The testing.variationName study setting doesn't seem to work.

### ISSUE THREAD

**Issue by wisniewskit** [mozilla/shield-studies-addon-utils#195]
In the console logs, it seems to select its own variation regardless of whether I specify a valid `testing.variationName` or not.

**Comment by gregglind** [comment]
issue is in studyUtils.js:setup function where I treat weightedVariations like k-v object, when it's an array.  also needs a test in browser.study.api.js where the testing variation is VERY VERY RARE, and yet is chosen.

**Comment by wisniewskit** [comment]
For local testing, I'm just changing that line to this:
```
studySetup.weightedVariations.find(
  ({name}) => name === studySetup.testing.variationName) ||
```

**Comment by motin** [comment]
Fixed - #197
